db/internal_stats.cc

```diff
@@ -230,7 +230,7 @@ static const std::string num_live_versions = "num-live-versions";
 static const std::string current_version_number =
     "current-super-version-number";
 static const std::string estimate_live_data_size = "estimate-live-data-size";
-static const std::string min_log_number_to_keep = "min-log-number-to-keep";
+static const std::string min_log_number_to_keep_str = "min-log-number-to-keep";
 static const std::string base_level = "base-level";
```

Comment:
maybe also rename `base_level` to `base_level_str` considering `base_level` is also a commonly used word in our codebase?